.gitignore

```diff
@@ -0,0 +1 @@
+*.java
```

Comment:
Please consider also adding .class and .kt

Reply:
:+1: